guides/common/modules/proc_configuring-dynamic-dns-update-with-tsig-authentication.adoc

```diff
@@ -25,7 +25,7 @@ To configure dynamic DNS update with TSIG authentication, complete the following
 
 . On the IdM Server, add the following to the top of the `/etc/named.conf` file:
 +
-[options="nowrap" subs="+attributes"]
+[source, none, options="nowrap" subs="+attributes"]
```

Comment:
```suggestion
[source,text]
```

Reply:
Same for other code blocks with output.

Reply:
I have never seen `[source,none]` used in d/s docs. AFAIK, we do not have any guidelines about syntax highlighting except not to use `[source,bash]` for CLI commands because it misinterprets `#` as a comment mark.

Reply:
Yes, the latter is one of the main points I am trying to fix.

Reply:
Using "none" is fine.